### ISSUE TITLE
About the data release

### ISSUE THREAD

**Issue by mengcaopku** [hshjerry/VideoEspresso#4]
Thanks for your excellent work. I want to know when will you release the full dataset including both the training and test parts. Thanks.

**Comment by hshjerry** [comment]
Thanks for your attention. We plan to release the full dataset before March. The test set and benchmark have been released, enjoy the Espresso!

**Comment by dle666** [comment]
Thanks for your excellent work. March is already here, will the dataset be released?

**Comment by hshjerry** [comment]
Sorry for delay :( We are still check the training set, but will be done not soon. Thanks for all guys' waiting and patience.

**Comment by hshjerry** [comment]
@mengcaopku @dle666 Hi, the training set has been updated! The video version is also uploading :)

**Comment by dle666** [comment]
Thank you so much! Looking forward to the video version of the data.

**Comment by Mac0q** [comment]
Will you consider upload the training script?

**Comment by hshjerry** [comment]
@mengcaopku @dle666 The video version has been updated also.
